app/src/main/res/values/strings.xml

```diff
@@ -76,8 +76,9 @@
     <string name="importExport">Import/Export</string>
     <string name="importName">Import</string>
     <string name="exportName">Export</string>
-    <string name="importExportHelp">Data is imported to/exported from BudgetWatch.csv on external storage</string>
+    <string name="importExportHelp">Data is imported to/exported from BudgetWatch.<xliff:g id="extension">%s</xliff:g> on external storage</string>
     <string name="importedFrom">Imported from: %1$s</string>
+    <string name="fileFormat">File format</string>
```

Comment:
<string name="importedFrom">Imported from: %1$s</string>
<string name="fileFormat">File format</string>


<string name="importiert von">Imported from: %1$s</string>
<string name="Dateiformat">File format</string>

here you have it, not too much to translate

Am 07.09.2016 um 01:20 schrieb Branden Archer:
> <string name="importedFrom">Imported from: %1$s</string>
> > +    <string name="fileFormat">File format</string>